### PR TITLE
Add release notes for AEPCore 3.2.0a and Android 1.8.2

### PR DESCRIPTION
**Pull request by nporter-adbe**

### CODE REVIEW

**Review by yangyansong-adbe — APPROVED** [review]

**Comment by nporter-adbe** [comment]
@calebk1 need a review here.

File: foundation-extensions/mobile-core/mobile-core-release-notes.md [hunk 1]
```diff
@@ -1,5 +1,33 @@
...
+* Support for handling identities request `Event`'s in AEPIdentity
+* Improve public visiblity of `RuleConsequence`
+* Added `getDeviceModelNumber` to `SystemInfoService`
+* Various additions to `ThreadSafeDictionary`
+* Added the ability to make a network request with raw data that is not UTF encoded
+* Fixed a bug where condition definitions that did not contain a value were not handled correctly
+* Introduced an API to set button image data to the `FloatingButton`
+* Added `optimize` `EventType`
+* Introduced an API to hide the `FullscreenMessage`
+* Fixed a bug where token (~ timestampu) was not expanded correctly
```
Comment (calebk1) [inline foundation-extensions/mobile-core/mobile-core-release-notes.md:28]: Is the variable called timestampu?

**Review by yangyansong-adbe — APPROVED** [review]

**Review by yangyansong-adbe — APPROVED** [review]

**Review by calebk1 — COMMENTED** [review]
Just one comment. Looks good otherwise!

File: foundation-extensions/mobile-core/mobile-core-release-notes.md (hunk 1 above)
Comment (nporter-adbe) [reply foundation-extensions/mobile-core/mobile-core-release-notes.md:28]: Yeah, thats correct.

**Review by calebk1 — APPROVED** [review]
LGTM!

**Pull request merged**